app-arch/unar/unar-1.10.8.ebuild

```diff
@@ -35,7 +35,7 @@ PATCHES=( "${FILESDIR}"/${P}-Wint-conversion.patch )
 
 check_objc_toolchain() {
 	if tc-is-gcc; then
-		has_version 'sys-devel/gcc[-objc]' &&
```

Comment:
Would this help instead?

`has_version 'sys-devel/gcc[-objc]:*'`

Reply:
Well that would allow the check to then pass with just one gcc version having the use flag enabled. You could then get a gcc version with it disabled past the check leading to it failing later.

Reply:
I guess we really need to just do a fortran.eclass style thing.